src/WheelsAndGoods.Application/Models/Paginations/PaginationBase.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace WheelsAndGoods.Application.Models.Paginations;
+
+public abstract class PaginationBase
+{
+    const int maxPageSize = 50;
+    public int PageNumber { get; set; } = 1;
+    private int _pageSize = 50;
+
+    public int PageSize
```

Comment:
Max value should be controlled on validator level not POCO/DTO etc. Moreover current logic is unclear, imagine that you are an external client that uses our API, and you pass PageSize=100 but got 50 items, you will think it's a bug.